i18next-scanner.config.js

```diff
@@ -0,0 +1,48 @@
+const { i18nextToPo } = require('i18next-conv');
+const VirtualFile = require('vinyl');
+
+module.exports = {
+  input: [
+    'src/**/*.{html,js}',
+  ],
+  options: {
+    contextSeparator: '|',
+    pluralSeparator: '|',
+    ns: [
+      'categories',
+      'footer',
+      'meal-planner',
+      'navigation',
+      'search',
+      'shopping-list',
+      'starred-recipes',
+    ],
+    resource: {
+      loadPath: 'public/locales/{{lng}}/{{ns}}.json',
```

Comment:
This has a dependency on the [webpack build](https://github.com/openculinary/frontend/blob/26cfb43779fc2ccd7cd3955f148a3c5cef6a760a/webpack.config.js#L23).